specs/botframework-protocol/channel-update-spec.md

```diff
@@ -0,0 +1,80 @@
+# Updating the botframework-channel swagger file
+
+## Abstract
+This document describe specific guidelines for upding the [swagger definiton file](https://github.com/microsoft/botframework-sdk/blob/master/specs/botframework-protocol/botframework-channel.json) hosted in this repo. These guidelines specify the steps and requirements to mainting a consistent source of truth that describes the state of connector-features in each SDK codebase. This source of truth is the combination of 2 concepts:
```

Comment:
spelling: **mainting**

Reply:
@EricDahlvang As a matter of "The easy way vs The Hard way" you can edit this directly yourself in the browser while doing a PR review. I've started doing that, just to fix the back-and-forth and make things go faster...

Reply:
@cleemullins k, that's definitely faster!